bees/twiliobee/twiliobee.go

```diff
@@ -0,0 +1,47 @@
+// Package twiliobee is a Bee that is able to send SMS messages.
+package twiliobee
+
+import (
+	twilio "github.com/carlosdp/twiliogo"
+	"github.com/muesli/beehive/bees"
+)
+
+// TwilioBee is a Bee that is able to send SMS messages.
+type TwilioBee struct {
+	bees.Bee
+
+	account_sid string
+	auth_token  string
+	from_number string
+	to_number   string
+}
+
+// Action triggers the action passed to it.
+func (mod *TwilioBee) Action(action bees.Action) []bees.Placeholder {
+	outs := []bees.Placeholder{}
+
+	switch action.Name {
+	case "send":
+		body := ""
+
+		action.Options.Bind("body", &body)
+
+		client := twilio.NewClient(mod.account_sid, mod.auth_token)
```

Comment:
Do we really need to create a new twilio client for every message? Maybe it's enough to create it once in Run()?